chain/network/src/routing/routing_table_actor.rs

```diff
@@ -316,7 +320,7 @@ impl RoutingTableActor {
         for peer_id in peers_to_remove.iter() {
             let _ = update.set_ser(
                 ColPeerComponent,
-                Vec::from(peer_id).as_ref(),
```

Comment:
Eg here, looking at all those allocated and then immediately freed vectors is painful :)